src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
The interface name `Draw` is a verb, which is not recommended for class/interface names. Consider using a noun or a noun phrase that describes the capability, such as `Drawable`.